### ISSUE TITLE
Active Cases and Recovered Cases

### ISSUE THREAD

**Issue by paolinic03** [CSSEGISandData/COVID-19#1180]
Since **recovery data** is inconsistent or null for various countries, **active cases** is also incorrect because active cases is just SUM(Confirmed Cases)-SUM(Closed Cases). So without accurate death and recovered cases, the only case type we can rely on at this time is Confirmed Cases which is the least important metric for tracking progress because there are likely 100s of thousands more positive cases out there that have not/ will not be tested including asymptomatic cases.

**Comment by jcavfen9** [comment]
I think it depends of the country. For some countries (China for instance), "Recovered" data seems consistent. For some other (France I know), it is inconsistent and false. Same think for "Confirmed" as this number is very reliant on the country testing policy. Germany and South Korea are testing aggressively so the Fatality Rate seems more correct than US or France that test only a few number of people. I happened to think that the most reliable data is death count (except for Iran). [My dashboard using the JHU data](https://public.tableau.com/views/Coronavirus2019-nCoVstats/Coronavirus2019-nCoV?:display_count=y&:origin=viz_share_link) .

**Comment by paolinic03** [comment]
Yea, you are probably right. Just so hard to tell. For those of us reporting specifically on the US, it has been tough to know where we stand.

Thanks for your thoughts.

**Comment by bobblee** [comment]
@jcavfen9 : How  are you calculating "Active" : confirmed - ( Deaths + Recovered) ?
I know with this data there are NO absolutes. However, it looks like you've looked a decent bit and may have thoughts. .

**Comment by paolinic03** [comment]
JHU dataset calculates active as

Active cases = total confirmed - total recovered - total deaths.

**Comment by jcavfen9** [comment]
> @jcavfen9 : How are you calculating "Active" : confirmed - ( Deaths + Recovered) ?

Yes. I don't get this data from the [JHU dataset](https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_daily_reports) I'm using. So I calculate it that way. But one must very cautious with this data as in my view both "Confirmed" and "Recocered" numbers are very testing policy dependant and countries have quite different testing policies.
